cluster/swarm/cluster.go

```diff
@@ -568,9 +568,9 @@ func (c *Cluster) CreateVolume(request *volume.VolumesCreateBody) (*types.Volume
 		wg.Wait()
 	} else {
 		config := cluster.BuildContainerConfig(containertypes.Config{Env: []string{"constraint:node==" + parts[0]}}, containertypes.HostConfig{}, networktypes.NetworkingConfig{})
-		nodes, err := c.scheduler.SelectNodesForContainer(c.listNodes(), config)
-		if err != nil {
-			return nil, err
+		nodes, errs := c.scheduler.SelectNodesForContainer(c.listNodes(), config)
+		if errs != nil {
+			return nil, errs
```

Comment:
@YuPengZTE Good catch! I don't want to introduce yet another variable to deal with this. How about removing `:=` like this? 
```
                var nodes []*node.Node
		nodes, err = c.scheduler.SelectNodesForContainer(c.listNodes(), config)
 		if err != nil {
 			return nil, err
```